docs/remote/ssh.md

```diff
@@ -19,33 +19,45 @@ This lets VS Code provide a **local-quality development experience** — includi
 
 ## Getting started
 
+### System Requirements
+
+**Local:** See minimum requirements for [VS Code](/docs/supporting/requirements.md). A supported [OpenSSH compatible SSH client](/docs/remote/troubleshooting.md#installing-a-supported-ssh-client) must also be installed.
+
+**Remote SSH Host**: A running [SSH server](/docs/remote/troubleshooting.md#installing-a-supported-ssh-server) on:
+
+- **Full support:** x86_64 Debian 8+, Ubuntu 16.04+, CentOS / RHEL 7+.
+
+- **Experimental support:** ARM32 Raspbian 8+ in [Visual Studio Code Insiders](https://code.visualstudio.com/insiders/) only.
+
+Other `glibc` based Linux distributions for x86_64 and ARM32 (in VS Code Insiders) should work if they have the needed prerequisites. See the [Remote Development and Linux](/docs/remote/linux.md) article for information prerequisites and tips for getting community supported distributions up and running.
+
+While experimental ARM32 support is available in [VS Code - Insiders](https://code.visualstudio.com/insiders/), some extensions installed on ARM32 devices may not work due the use of x86 native code in the extension.
+
 ### Installation
 
 To get started you need to:
 
-1. Make sure your local OS meets the minimum requirements for [VS Code](/docs/supporting/requirements.md) and the Linux host you want to connect to [has the needed prerequisites](/docs/remote/linux.md).
-
-2. Install an [OpenSSH compatible SSH client](/docs/remote/troubleshooting.md#installing-a-supported-ssh-client) if one is not already present.
+1. Install an [OpenSSH compatible SSH client](/docs/remote/troubleshooting.md#installing-a-supported-ssh-client) if one is not already present.
 
     > **Note:** PuTTY is not supported on Windows since the `ssh` command must be in the path.
 
-3. Install [Visual Studio Code](https://code.visualstudio.com/).
+2. Install [Visual Studio Code](https://code.visualstudio.com/) or [Visual Studio Code Insiders](https://code.visualstudio.com/insiders/).
 
-4. Install the [Remote Development](https://aka.ms/vscode-remote/download/extension) extension pack.
+3. Install the [Remote Development](https://aka.ms/vscode-remote/download/extension) extension pack.
 
-5. [Optional] If your server requires multi-factor authentication, set `"remote.SSH.showLoginTerminal": true` in `settings.json` and enable the `ControlMaster` SSH feature. [See here for details](/docs/remote/troubleshooting.md#enabling-alternate-ssh-authentication-methods).
+4. [Optional] If you need to enter a token or password when connecting to your host, [you can enable `ControlMaster` in your SSH config](/docs/remote/troubleshooting.md#enabling-alternate-ssh-authentication-methods) to prevent you from having to enter it multiple times. However, we have seen mixed results with this setting and Windows SSH clients.
```

Comment:
Does it work with any windows client? I think it's not supported on windows at all.

Reply:
Yeah I wasn't 100% sure, so I didn't lock in, but if you haven't heard of one either let's just say it's not supported.